src/scale.js

```diff
@@ -133,6 +134,10 @@ export class Scale {
         var b=0;
 
         switch(color) {
+            case 'NetSage':
```

Comment:
Please pick a more generic name for the color scheme.

Reply:
I will do it today! Thanks